README.md

```diff
@@ -193,6 +200,16 @@ feed:
         - updates
 ```
 
+Or pass `categories: true` to generate a feed for all categories:
+
+```yml
+feed:
+  collections:
+    changes:
+      path: "/changes.atom"
+      categories: true
```

Comment:
Can we have a test for this? It seems like we could over-generate. That is we could generate a category feed which has zero entries since a collection doesn't have any documents with a category that is only present in another collection.

1. Collection "posts" has categories "A" and "B"
2. Collection "archives" has categories "C"
3. Will we generate `/feed/archives/A.xml` with zero entries?